chain-watch/proto/watch.proto

```diff
@@ -0,0 +1,60 @@
+syntax = "proto3";
+
+package iohk.chain.watch;
+
+
+message Block {
+  bytes content = 1;
+}
+
+message BlockSubscriptionRequest {}
+
+message TipSubscriptionRequest {}
+
+message MempoolSubscriptionRequest {}
+
+message SyncMultiverseRequest {
+  // send blocks with this chain_length or greater
+  uint32 from = 1;
+}
+
+message BlockId {
+  bytes content = 1;
+}
+
+message MempoolEvent {
+    bytes fragment_id = 1;
+    oneof event {
+        MempoolFragmentInserted inserted = 2;
+        MempoolFragmentRejected rejected = 3;
+        MempoolFragmentInABlock in_a_block = 4;
+    };
+}
+
+
+message MempoolFragmentInserted {}
+
+message MempoolFragmentRejected {
+    string reason = 1;
+}
+
+message MempoolFragmentInABlock {
+    BlockId block = 1;
+}
+
+
+service SubscriptionService {
+  // get a stream of blocks succesfully processed by the node, this means they
+  // are already validated.
+  // the parent of a block will always be streamed before the block itself.
+  rpc BlockSubscription(BlockSubscriptionRequest) returns (stream Block);
+
+  // get tip updates
+  rpc TipSubscription(TipSubscriptionRequest) returns (stream BlockId);
+
+  rpc MempoolSubscription(MempoolSubscriptionRequest) returns (stream MempoolEvent);
```

Comment:
And now I am not sure that this should belong to chain-libs. We usually did not introduce the concept of mempool at the chain-libs level.

Reply:
Well, current plan is to move it to chain-network, actually. If we put this in jormungandr... And we want to re-use from a different repository, then we would have to include jormungandr there somehow?

Although, @mzabaluev motivation to put it in `chain-network` was the dependencies/build process, I think.

Reply:
> motivation to put it in `chain-network` was the dependencies/build process, I think

Yeah, mostly that I think.

> then we would have to include jormungandr there somehow

Not a huge problem, we are doing that for `jormungandr-lib` already.